docs(tile-converter): I3S Server (#2564)
visgl/loaders.gl: docs/modules/tile-converter/cli-reference/tile-converter.md

| instant-node-writing      | \*                        |                           | Whether the converter should keep JSON resources ([3DNodeIndexDocuments](https://github.com/Esri/i3s-spec/blob/master/docs/1.8/3DNodeIndexDocument.cmn.md) and [nodePages](https://github.com/Esri/i3s-spec/blob/master/docs/1.8/nodePage.cmn.md)) on disk during conversion. The default behavior is the converter keeps JSON resources in memory till the end of conversion. Those resources need to be updated during conversion (adding child nodes and neighbor nodes). If this option is set `true` the converter will keep JSON resources on disk all the time. Use this option for large datasets when the nodes tree is large and "memory overflow" error occurs. Instant node writing saves memory usage in cost of conversion speed (>2 times slower). |

> | split-nodes               | \*                        |                           | Prevent to merge similar materials that could lead to incorrect visualization. By default, the converter tries to merge PBR materials to create one node for all primitives in a glTF file. Using this option the converter will generate new node for every PBR material in the glTF file. With this option the resulting layer usually has refinement issues because "leaf" nodes are generated in the middle of the nodes tree                                                                                                                                                                                                                                                                                                                                 |

Reviewer: Nit: try to keep descriptions short by cutting unnecessary words
```suggestion
| split-nodes               | \*                        |                           | The converter will generate new node for every PBR material in the glTF file. Prevent merge of similar materials that could lead to incorrect visualization. By default, the converter tries to merge PBR materials to create one node for all primitives in a glTF file. Note can cause refinement issues because "leaf" nodes are generated in the middle of the nodes tree                                                                                                                                                                                                                                                                                                                                 |
```
